extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -136,28 +140,39 @@ export class AmpUserNotification extends AMP.BaseElement {
   }
 
   /**
+   * Constructs the new href to execute a `GET` request with the
+   * `elementId` and `ampUserId` query params appended.
+   * @param {string} ampUserId
+   * @return {!Promise<string>}
+   * @private
+   */
+  buildGetHref_(ampUserId) {
+    return this.urlReplacements_.expand(this.showIfHref_).then(href => {
+      const hrefObj = parseUrl(href);
+      const fields = `elementId=${encodeURIComponent(this.elementId_)}&` +
```

Comment:
Question here is whether or not this is part of the customizable API or you want to force it. Depends on the problem at hand.